usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+
+def test_sles_hardened(host, get_release_value, is_sles_sap, swap_file):
```

Comment:
If we keep it like this with a fixture there's a typo: s/swap_file/swapfile. Also the fixture never gets called.

Reply:
Fixed.